app/src/main/java/com/dogandpigs/fitnote/core/FNApplication.kt

```diff
@@ -0,0 +1,11 @@
+package com.dogandpigs.fitnote.core
+
+import android.app.Application
+import dagger.hilt.android.HiltAndroidApp
+
+@HiltAndroidApp
+class FNApplication: Application() {
+    override fun onCreate() {
+        super.onCreate()
+    }
+}
```

Comment:
```suggestion
@HiltAndroidApp
class FNApplication: Application()
```
현재 코드를 보면 따로 구현된게 없는것 같은데 이렇게 하면 어떨까요?

Reply:
네넵 미리 만들어놨었습니다 ㅋ_ㅋ